test/test_package_loading.py

```diff
@@ -0,0 +1,50 @@
+import os
+from pathlib import Path
+import time
+
+from http.server import HTTPServer, SimpleHTTPRequestHandler
+import threading
+
+import pytest
+
+
+@pytest.fixture
+def static_web_server():
+    # serve artefacts from a different port
+    build_dir = Path(__file__).parent.parent / 'build'
+    os.chdir(build_dir)
+    try:
+        url, port = ('127.0.0.1', 8888)
```

Comment:
This assumes that port `8888` is unused, not sure to what extent that may be an issue.

Reply:
It becomes an issue when the test process doesn't shut down cleanly -- then the port remains reserved by the system for a few seconds and can't be opened again.